cmd/wallet-sdk-js/jsinterop/agent.go

```diff
@@ -177,6 +202,23 @@ func getSubmissionRequirements(_ js.Value, args []js.Value) (any, error) {
 	return util.MapTo(req, types.SerializeMatchedSubmissionRequirement)
 }
 
+func verifyCredentialsStatus(_ js.Value, args []js.Value) (any, error) {
+	if agentInstance == nil {
+		return nil, walleterror.NewExecutionError(
```

Comment:
Should use the new "InvalidSDKUsage" method to generate these types of error codes

Reply:
`walleterror.NewInvalidSDKUsageError`